### ISSUE TITLE
Running relayer (v2.0.0, with `-p events`) against existing connection consistently throws an error

### ISSUE THREAD

**Issue by joe-bowman** [cosmos/relayer#899]
I have two chains (qstest-1 and lstest-1) and use Hermes to establish a connecion, and a ics20 channel.

Relayer is configured to run with the following config:

```
global:
    api-listen-addr: :5183
    timeout: 10s
    light-cache-size: 20
chains:
  qstest-1:
      type: cosmos
      value:
        key: testkey
        chain-id: qstest-1
        rpc-addr: http://quicksilver:26657
        account-prefix: quick
        keyring-backend: test
        gas-adjustment: 1.5
        gas-prices: 0.0uqck
        debug: true
        timeout: 180s
        output-format: json
        sign-mode: direct
  lstest-1:
      type: cosmos
      value:
        key: testkey
        chain-id: lstest-1
        rpc-addr: http://testzone1-1:26657
        account-prefix: cosmos
        keyring-backend: test
        gas-adjustment: 1.5
        gas-prices: 0.0uqck
        debug: true
        timeout: 180s
        output-format: json
        sign-mode: direct
paths:
    demo:
        src:
            chain-id: qstest-1
            client-id: 07-tendermint-0
            connection-id: connection-0
        dst:
            chain-id: lstest-1
            client-id: 07-tendermint-0
            connection-id: connection-0
        src-channel-filter:
            rule: ""
            channel-list: []
```

Running relayer with the above config (chain IDs, clients, connections, endpoints, etc. are correct) endlessly yields the following error:

```
relayer_1       | ts=2022-08-05T11:08:29.306724Z lvl=warn msg="No cached IBC header for client trusted height" chain_id=lstest-1 client_id=07-tendermint-0 height=39
```

**Comment by jackzampolin** [comment]
Thanks for the report @joe-bowman will take a look today

**Comment by agouin** [comment]
> lvl=warn msg="No cached IBC header for client trusted height"

In order to assemble `MsgUpdateClient` messages, we need the validator set from (height + 1) of the client state's consensus height. Since the `PathProcessor` runs after any new information is available from either chain, it is common that we will observe a `MsgUpdateClient` with a new consensus height before that consensus height + 1 is observed on the counterparty. The `PathProcessor` process will then run again and be able to assemble and send a new `MsgUpdateClient`.

TL;DR: This is a benign warning, and we can probably change the log level to debug.

**Comment by jtieri** [comment]
Looks like the log level for this got changed from `WARN` to `DEBUG` in #843 so I'm gonna close this issue out.
